### PR TITLE
Will

### PR DESCRIPTION
**Pull request by WilliamNguyen21**
Create New Playlist Page Without Playlist Image

### CODE REVIEW

File: routes/api.js [hunk 1]
```diff
@@ -7,12 +7,17 @@ import path from 'path'
...
 import multer from "multer";
+import cors from 'cors';
 dotenv.config();
 
 const router = express.Router();
 const SECRET_KEY = process.env.ACCESS_TOKEN_SECRET;
 const userRoles = { "Listener": 1, "Artist": 2, "Admin": 3 }
 
+router.use(cors({
+  origin: 'http://localhost:5173', // Your React app URL
```
Comment (jo5huar3) [inline routes/api.js]: remove this

File: routes/api.js [hunk 2]
```diff
@@ -7,12 +7,17 @@ import path from 'path'
 import fs from 'fs';
 import { fileURLToPath } from "url";
 import multer from "multer";
+import cors from 'cors';
```
Comment (jo5huar3) [inline routes/api.js]: remove

**Review by jo5huar3 — CHANGES_REQUESTED** [review]
I added a couple things to delete. Things like cors  are already defined

**Review by WilliamNguyen21 — COMMENTED** [review]
fixed create playlist page, and working library page,
- stilll working on images for playlist

**Pull request merged**